.github/ISSUE_TEMPLATE/issue-report.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Issue report"
+description: Report documentation issues such as inaccuracies, broken links, typos, or missing information.
+title: "[Issue]: "
+labels: ["Issue"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi 👋. Thanks for taking the time to fill out this issue report!
+        This form will create an issue that Elastic's docs team will triage and prioritize.
+        You can also open a PR instead.
+  - type: dropdown
+    attributes:
+      label: Type of issue
+      description: What type of issue are you reporting?
+      multiple: false
+      options:
+        - Not clear
+        - Missing information
+        - Could be improved
+        - I can't find what I'm looking for
+  - type: input
+    id: link
+    attributes:
+      label: What documentation page is affected
+      description: Include a link to the page where you're seeing the problem.
+    validations:
+      required: true
```

Comment:
I'd love to auto-fill this in with the page the user clicked on the link from. For example, in the [Docker Docs](https://docs.docker.com/build/), if you click "Request changes" from https://docs.docker.com/build/, this happens:

<img width="724" alt="Screenshot 2025-03-21 at 10 31 25 AM" src="https://github.com/user-attachments/assets/a06dc18a-2092-4624-baf3-d93757bc8a6a" />

Looks like it's just a query string to make this happen:

```
https://github.com/docker/docs/issues/new?template=doc_issue.yml&location=https%3a%2f%2fdocs.docker.com%2fbuild%2f&labels=status%2Ftriage
```